docs/user/faq.rst

```diff
@@ -160,6 +160,56 @@ try one of the generic
 If you use an API gateway, you might also look into what CORS functionality
 it provides at that level.
 
+Why is my request with authorization blocked despite ``cors_enable``?
+---------------------------------------------------------------------
+
+When you are making a cross-origin request from the browser (or another HTTP
+client verifying CORS policy), and the request is authenticated using the
+Authorization header, the browser adds ``authorization`` to
+Access-Control-Request-Headers in the preflight (``OPTIONS``) request,
+however, the actual authorization credentials are omitted at this stage.
+
+If your request authentication/authorization is performed in a
+:ref:`middleware <middleware>` component which rejects requests lacking
+authorization credentials by raising an instance of :class:`~.HTTPUnauthorized`
+(or rendering a 4XX response in another way), a common pitfall is that even an
+``OPTIONS`` request (which is lacking authorization as per the above
+explanation) yields an error in this manner. As a result of the failed
+preflight, the browser chooses not proceed with the main request.
+
+If you have implemented the authorization middleware yourself, you can simply
+let ``OPTIONS`` pass through:
+
+.. code:: python
+
+    class MyAuthMiddleware:
+        def process_request(self, req, resp):
+            # NOTE: Do not authenticate OPTIONS requests.
+            if req.method == 'OPTIONS':
+                return
+
+            # -- snip --
+
+            # My authorization logic...
+
+Alternatively, if you rather prefer not to fiddle with third party middleware,
+you can subclass it:
```

Comment:
as is it does not seem clear. How about

> Alternatively, if you use a third party middleware that does not offer that options,
you can subclass it:

Reply:
I agree my sentence isn't very clear... 

However, third party middleware not offering that option sounds somewhat strange too. Unless it is written in C and delivered compiled, you can well edit its code too :thinking:

Reply:
it's not usually practical to edit a library. Maybe I don't understand what you mentioned

Reply:
> it's not usually practical to edit a library.

That's exactly what I was referring to. I'll try to reword to reflect just that.